docs/usage.md

```diff
@@ -16,10 +18,11 @@ Sarek is designed to handle single samples, such as single-normal or single-tumo
 The typical command for running the pipeline is as follows:
 
 ```bash
-nextflow run nf-core/sarek --input samplesheet.csv --outdir <OUTDIR> --genome GATK.GRCh38 -profile docker
+nextflow run nf-core/sarek --input ./samplesheet.csv --outdir ./results --genome GATK.GRCh38 --tools <TOOLS> -profile docker
```

Comment:
I see `<OUTDIR>` to `./results`  but now we have `--tools <TOOLS>` - should we perhaps do `--tools strelka` or `--tools freebayes` instead?

Reply:
I agree with Rike on this